packages/core/integration-tests/test/integration/_test/.parcelrc

```diff
@@ -0,0 +1,6 @@
+{
+  "extends": "@parcel/config-default",
+  "transformers": {
+    "*.js": ["parcel-transformer-passthrough", "..."]
```

Comment:
The test passes if **either**:
- The pattern is updated to:
```suggestion
    "*.{js,mjs}": ["parcel-transformer-passthrough", "..."]
```

- or the additional transformer is excluded:
```suggestion
    "*.js": ["..."]
```